src/node_modules/logger.js

```diff
@@ -172,13 +160,21 @@ export default class Logger extends Writable {
 		this[$reconnectInitialDelay]    = opts.reconnectInitialDelay || defaults.reconnectInitialDelay;
 		this[$reconnectMaxDelay]        = opts.reconnectMaxDelay || defaults.reconnectMaxDelay;
 		this[$reconnectBackoffStrategy] = opts.reconnectBackoffStrategy || defaults.reconnectBackoffStrategy;
+		this.debug                      = opts.debug || false;
+		
+		if (!this.debug) {
+			//if there is no debug set, empty logger should be used
+			this.debugLogger = { log: () => { } };
```

Comment:
Since emptyDebugLogger is defined in defaults.js, it seems like it should be used here. Or alternatively just remove emptyDebugLogger from defaults.js if it's not going to be used anywhere.